src/Sarif.UnitTests/Transformers/SarifVersionOneToCurrentTests.cs

```diff
@@ -1,16 +1,41 @@
 ﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
 // license. See LICENSE file in the project root for full license information.
 
+using System.IO;
+using System.Reflection;
 using FluentAssertions;
 using Microsoft.CodeAnalysis.Sarif.VersionOne;
 using Microsoft.CodeAnalysis.Sarif.Visitors;
 using Newtonsoft.Json;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.Sarif
+namespace Microsoft.CodeAnalysis.Sarif.UnitTests.Transformers
 {
     public class SarifVersionOneToCurrentTests
     {
+        private static readonly Assembly ThisAssembly;
+        private static readonly string OutputFolderPath;
+        private const string TestLogResourceNameRoot = "Microsoft.CodeAnalysis.Sarif.UnitTests.Transformers.TestLogs";
+
+        static SarifVersionOneToCurrentTests()
+        {
+            ThisAssembly = Assembly.GetExecutingAssembly();
+            OutputFolderPath = Path.Combine(Directory.GetParent(Path.GetDirectoryName(ThisAssembly.Location)).FullName, "UnitTestOutput");
+
+            if (!Directory.Exists(OutputFolderPath))
+            {
+                Directory.CreateDirectory(OutputFolderPath);
+            }
+            else
+            {
+                string[] files = Directory.GetFiles(OutputFolderPath);
+                foreach (string path in files)
+                {
+                    File.Delete(path);
+                }
+            }
```

Comment:
>} [](start = 12, length = 1)

Rather than delete the files by hand, you can do this (and then you can create the directory unconditionally):
```
if (Directory.Exists(OutputFolderPath)
{
    Directory.Delete(OutputFolderPath, recursive: true);
}

Directory.CreateDirectory(OutputFolderPath);
```